src/main/java/org/openrewrite/java/testing/junitassertj/AssertNullToAssertThat.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.junitassertj;
+
+import org.openrewrite.AutoConfigure;
+import org.openrewrite.java.*;
+import org.openrewrite.java.tree.*;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.openrewrite.Formatting.EMPTY;
+import static org.openrewrite.Formatting.format;
+import static org.openrewrite.Tree.randomId;
+import static org.openrewrite.java.tree.MethodTypeBuilder.newMethodType;
+
+/**
+ * This is a refactoring visitor that will convert JUnit-style assertNull() to assertJ's assertThat().isNull().
+ *
+ * This visitor only supports the migration of the following JUnit 5 assertNull() methods:
+ *
+ * <PRE>
+ *     assertNull(Object actual) -> assertThat(condition).isNull()
+ *     assertNull(Object actual, String message) -> assertThat(condition).as(message).isNull();
+ *     assertNull(Object actual, Supplier<String> messageSupplier) -> assertThat(condition).withFailMessage(messageSupplier).isNull();
+ * </PRE>
+ */
+@AutoConfigure
+public class AssertNullToAssertThat extends JavaIsoRefactorVisitor {
```

Comment:
This is pretty close to being a copy+paste of AssertNotNullToAssertThat. 
Could they be merged into a single visitor without introducing too much complexity? Or perhaps introduce a utility class which handles all of the functionality common between these assertj visitors.

Reply:
Based upon our conversation this morning, I'm not going to refactor these visitors for now, but will keep this in mind for future visitors.